src/layouts/partial-cards.html

```diff
@@ -295,8 +295,8 @@
       return routeMatch ? routeData.view : '';
     }
 
-    computeTitle(views, locationName) {
-      return views && views.length > 0 ? 'Home Assistant' : locationName;
+    computeTitle(locationName) {
+      return locationName || 'Home Assistant';
```

Comment:
So let's take a step back here and let's see why the feature is what it is, before throwing PRs at it.

 - The location name in the default Home Assistant config is "Home".
 - The default name for the default view is… the location name.

This means that adding views while not adding a default view resulted in seeing the location name twice, which is odd. That's why we decided to show "Home Assistant" instead of the location name, to prevent duplication.

In your PR you've changed the default title for the default view to no longer be the location name but instead be "Home". However, this is exactly the same as the default location name and so it would cause duplication again.

Reply:
Well config says `# Name of the location where Home Assistant is running`. So for me that's not the name for the default view.

This is how it looks now:
![change](https://user-images.githubusercontent.com/11984118/32639182-e1c2f102-c5c2-11e7-896e-5c7c84b2cc25.PNG)

Before: "test" was "Home Assistant" (but only if you have 2+ views) and "Home" was "homeassistant.name"

Reply:
The problem with using "Home" is that it is ambiguous. We mean that it's the first / landing page of states. However, we're also automation software for homes. That's why the location name was used.

So I'm not against this change overall. It makes sense. However, let's try to avoid the user having to see "Home" twice when it follows default instructions out of the box.

Reply:
@balloob 
Ok I see you point: if "default_view" does not exist in group config we see the name twice